### ISSUE TITLE
Bedrock Rag is missing ppt, pptx text splitting support with knowledgebase queries

### ISSUE THREAD

**Issue by converseKarl** [aws/aws-sdk-js-v3#6475]
### Describe the feature

Current support but is behind other systems at moment,

format: "pdf" || "csv" || "doc" || "docx" || "xls" || "xlsx" || "html" || "txt" || "md", // required

but in rag you need ppt, pptx,   (powerpoint splitting)

to 100% complete you need 
mp4, mp3, youtube URL, youtube channel and JSON (someone implied its in there but i've not seen it)

### Use Case

You have everything else except powerpoint, you have word, excel, txt, csv, html but no powerpoint.

A lot of information is in powerpoints, company info, results, and numerous presentations for training so ragifying them and using the information is quite a substantial set of user cases. 

### Proposed Solution

implement the embedding extraction from powerpoints (like you do with PDF's). If your using langchain in the background, its  5 minute job to add the PPT/PPTX conversion as a loader type but I don't know your underlying implementation.

### Acknowledgements

- [ ] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### SDK version used

3.651.1

### Environment details (OS name and version, etc.)

Linux Debian, Nodejs / EC2 or even using Lamba AWS direct

**Comment by Gab1988** [comment]
This is a big gap in the functionality - it is quite a simple addition. Anthropic supports it already so at least Claude models should be able to support pptx very easily
Thanks!

**Comment by converseKarl** [comment]
> This is a big gap in the functionality - it is quite a simple addition. Anthropic supports it already so at least Claude models should be able to support pptx very easily Thanks!

Indeed, but text spliting happens before any LLM query, but any model including titan should be able to use it easily, same as PDF's.

**Comment by zshzbh** [comment]
Thanks for the feedback! 
AWS SDKs uses the exported models from the service API, but AWS service API doesn't support ppt&pptx yet - 
https://docs.aws.amazon.com/bedrock/latest/APIReference/API_runtime_ContentBlock.html
https://docs.aws.amazon.com/bedrock/latest/APIReference/API_runtime_DocumentBlock.html

I think this is a good addition to the bedrock service API. I will move this issue to a cross SDK issue and open a feature request to the service team!

Thanks!
Maggie
